content/_data/changelogs/weekly.yml

```diff
@@ -1441,6 +1441,74 @@
       references:
         - url: https://jenkins.io/security/advisory/2017-11-08/
           title: security advisory
+- version: "2.90"
+  date: 2017-11-12
+  changes:
+    - type: major bug
+      message: >
+        Recover from legacy user configuration folders with <code>$</code> characters that are not part of hex escape sequences. (Regression in 2.89)
+      issue: 47909
+      pull: 3134
+    - type: major bug
+      message: >
+        Fix <em>Download from java.sun.com</em> installation method for JDK for downloads requiring an Oracle login after change to the Oracle site (again).
+      pull: 3136
+      issue: 47448
+    - type: rfe
+      message: >
+        Update Remoting from 3.13 to 3.14 in order to apply various performance and stability improvements.
+      pull: 3138
+      references:
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#314
+          title: full changelog
+        - issue: 37566
+        - issue: 45294
+        - issue: 47425
+        - issue: 47901
+        - issue: 47942
+    - type: rfe
+      message: >
+        Add option to trim specified string parameter values.
+      issue: 47115
+      pull: 3106
+    - type: rfe
+      message: >
+        Update Windows Agent Installer from 1.9.1 to 1.9.2:
+        Do not try to update <code>jenkins-slave.exe</code> on Unix agents when they connect.
+      pull: 3130
+      references:
+        - url: https://github.com/jenkinsci/windows-slave-installer-module/blob/master/CHANGELOG.md#192
+          title: full changelog
+        - issue: 47015
+    - type: rfe
+      message: >
+        Add a note explaining that the nodes count in label selection does not take into account permissions or other plugins.
+      issue: 47940
+      pull: 3137
+    - type: rfe
+      message: >
+        Improve Chinese localization.
+      pull: 3127
+    - type: rfe
+      message: >
+        Updated default workspace directory naming for new installations to use non-deprecated variable.
```

Comment:
Make the variable name explicit? And maybe it makes sense to add documentation link. I have not found docs in Wiki, but there is a built-in doc: https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

Reply:
Addressed this, including a reference to the issue.